pkg/amber/provenance.go

```diff
@@ -108,6 +108,11 @@ func GetBuildCmd(predicate slsav02.ProvenancePredicate) ([]string, error) {
 	return buildConfig.Command, nil
 }
 
+// GetBuilder extracts and returns the builder from the given ProvenancePredicate.
+func GetBuilder(predicate slsav02.ProvenancePredicate) string {
+	return predicate.Builder.ID
```

Comment:
nit: Why is this wrapped in a function here, but not in the case of `SLSAv02`? Could not this be inlined where it is called?